src/container/api.jl

```diff
@@ -60,3 +60,53 @@ end
 Used by the agent to get the protocol addr part
 """
 function protocol_addr(container::ContainerInterface) end
+
+function start(container::ContainerInterface) end
+function shutdown(container::ContainerInterface) end
+
+"""
+    run_mango(runnable, container_list)
+
+Run a Mango.jl agent system using the container included in `container_list` and therefor,
+the agents included in these containers. The runnable defines what the agent system shall
```

Comment:
`Run a Mango.jl agent system using the container included in `container_list` and therefor,
the agents included in these containers.`

This makes it very blurry what is what in Mango. The Containers are not the agent system and this just runs the containers.
And claiming it runs the agents (while I know what you mean, of course) could mislead people to think this calls some run(agent) function they have written or something.

Also:
"therefor" => "therefore".

Reply:
reworded